modules/java_api/cpp/ov_compiled_model.cpp

```diff
@@ -0,0 +1,27 @@
+#include <jni.h> // JNI header provided by JDK
```

Comment:
please name as just `compiled_model.cpp`

Reply:
Should I do this for all .cpp files of the new API? Some names are the same as for the old API.

Reply:
If so, can we create separate folders for old and new APIs?